README.md

```
@@ -2,313 +2,1413 @@
+- [Sathwik Yellapragada](https://www.sathwiky.dev) [Full-Stack Developer]
+- [Satish Jhanwer](https://satishjhanwer.github.io)
+- [Satish Vaishnav](http://satishvaishnav.in)
+- [Satyam Gupta](https://imlolman.github.io) ([@Imlolman](https://github.com/imlolman))
+- [Saurabh Daware](https://www.saurabhdaware.in)
+- [Saurabh Nemade](https://www.nemade.eu) (Staff Engineer)
+  ([@Saurabhnemade](https://github.com/saurabhnemade))
+- [Saurabh Patil](https://saurabhpatil.netlify.app) [Code Craftsman]
+- [Saurav M H](https://sauravmh.com)
+- [Savan Rathod](https://portfolio-8xwk.vercel.app)
+- [Sawyer Hollenshead](https://www.sawyer.soy)
+- [Sayan Adhikary](https://www.sayanadhikary.me) [Full Stack Mern] (https://github.com/sayanadhi03)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)